client/src/components/PostItem/PostItem.component.tsx

```diff
@@ -8,7 +8,7 @@ import EditButton from '../EditButton/EditButton.component'
 const PostItem = ({
   post: { id, title, tags, agencyId },
 }: {
-  post: BasePostDto
+  post: BasePostDto | { id: number; title: string; tags: []; agencyId: number }
```

Comment:
what's going on here?

Reply:
Because the search results returned by the search query is able to provide the `id`, `title` and `agencyId` but not the other properties in `BasePostDto`. However, an object with `BasePostDto` is passed into `PostItem` from `PostList`. Not sure how to do a 'minimal keys' thing so I just modified it to accept both types. 

Or would it be better to modify the object passed in from `PostList`?

Reply:
As discussed, we'll use `Pick` :)